test/.eslintrc

```diff
@@ -12,7 +12,8 @@
   "env": {
     "browser": false,
     "mocha": true,
-    "node": true
+    "node": true,
+    "es6": true
```

Comment:
what does this do?

Reply:
`new Map()` was giving an error in vs code, as it's supported in es6 onwards, so I updated `.eslintrc` to use `es6`

Reply:
![image](https://user-images.githubusercontent.com/8705386/129480200-191de818-948c-4f4a-9160-d40e2a95db54.png)